styles/find-and-replace.less

```diff
@@ -166,6 +166,25 @@ atom-workspace.find-visible {
       margin-right: @component-padding;
     }
   }
+
+  .wrap-icon {
+    @wrap-size: 96px;
```

Comment:
Done https://github.com/atom/find-and-replace/commit/386cbcb7ccf93fae9d4d25513d5f368baa00a211#diff-0c91d4121c1775ed1f0c74e547086288R171

I had to use modulus to keep the octicon a multiple of 16px.